options/options.go

```diff
@@ -82,14 +105,23 @@ toplevel:
 				case "p":
 					opts.ParseTime = true
 				case "c":
-					if in == len(osArgs)-1 {
-						break toplevel
+					opts.Config, err = readNextArg(in, false)
+					if err == notEnoughArgsErr {
+						break
```

Comment:
```suggestion
						continue
```

Reply:
Why continue? Shouldn't we have a crash flag or something instead when failing to parse this?

Reply:
Actually, yeah, it should be `break`, but breaking there won't do what we want, because it's a nested loop. Have to label the parent loop and `break <label>` from the nested one.